src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -70,35 +71,14 @@ public AlgorithmId(uint flags)
         // From ECMAKey.h
         private static readonly ImmutableArray<byte> s_ecmaKey = ImmutableArray.Create(new byte[] { 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0 });
 
-        // From strongname.h
-        // Note: different from the PUBLICKEYBLOB specified in wincrypt.h. The 
-        // PublicKey in the SnPublicKeyBlob contains a PUBLICKEYBLOB
-        //
-        // The public key blob has the following format as a little-endian packed C struct:
-        //
-        [StructLayout(LayoutKind.Sequential, Pack = 1)]
-#pragma warning disable 0649
-        private unsafe struct SnPublicKeyBlob
-        {
-            public UInt32 SigAlgId;          // Signature algorithm ID
-            public UInt32 HashAlgId;         // Hash algorithm ID
-            public UInt32 PublicKeySize;     // Size of public key data in bytes, not including the header
-            // Note: PublicKey is variable sized
-            public fixed byte PublicKey[1];  // PublicKeySize bytes of public key data
-        }
+        private const int SnPublicKeyBlobSize = 13;
 
         // From wincrypt.h
         private const byte PublicKeyBlobId = 0x06;
         private const byte PrivateKeyBlobId = 0x07;
 
         // internal for testing
-        internal unsafe static readonly int s_publicKeyHeaderSize = sizeof(SnPublicKeyBlob) - 1;
-
-        private static uint ToUInt32(ImmutableArray<byte> bytes, int offset)
-        {
-            Debug.Assert((bytes.Length - offset) > sizeof(int));
-            return (uint)(bytes[offset] | (bytes[offset + 1] << 8) | (bytes[offset + 2] << 16) | (bytes[offset + 3] << 24));
-        }
+        internal const int s_publicKeyHeaderSize = SnPublicKeyBlobSize - 1;
```

Comment:
I didn't introduce the field and as it has comment used by testing I don't know what if any internal tests can break due to your suggested renaming